UXWColorClass.js

```diff
@@ -0,0 +1,47 @@
+function UXWColorClass(props) {
+  return (
+    <div className="container">
+      <div className="flex flex-row">
+        <div className="flex flex-col">
+          <p className="pb-4 max-w-prose">
+            Brand colors are part of uxwcore.css. Use the color’s class name to update a HTML elements color.
+          </p>
+        </div>
+        <div className="flex-grow"></div>
+      </div>
+      {/* <UXWSubsection anchor="color-class" title="Background Color"> */}
+        <div className="flex max-w-prose"></div>
+        <div className="max-w-7xl grid grid-cols-2 gap-8 my-10 rounded-b-lg">
+          <div className="uxw-bg-acid-2" style={{position:"relative"}}>
+           <p style={{ color: "#152145", margin:"0", position:"absolute", top:"50%", left:"50%", transform:"translate(-50%, -50%)" }}>Hello Wizards!</p>
+          </div>
+          <div>
+            <UXWCodePanelHTMLCSS>
+              <div>
+                {`
+<div class="uxw-bg-acid-2">
+  <p>Hello Wizards!</p>
+</div>
+`}
+              </div>
+              <div>
+                {`
+.uxw-bg-acid-2 {
+  background-color: var(--uxw-color-merlins-beard-3);
+}`}
+              </div>
+            </UXWCodePanelHTMLCSS>
+          </div>
+        </div>
+        <p className="py-10">
+        CSS Color variables is set in 
+          <a href="https://github.com/UX-Wizards/design-system-web/blob/main/uxwcore.css" target="_blank">
```

Comment:
![image](https://user-images.githubusercontent.com/520254/150888806-90b90b74-8f72-4abf-8d3f-02e79e6e9555.png)
You might need a `&nbsp;` or `{ }`

Reply:
updated